src/js/dataMapping/search/awardType.js

```diff
@@ -76,10 +76,18 @@ export const awardTypeGroups = {
     other: ['09', '11']
 };
 
+export const analyticsAwardTypeGroupLabels = {
+    contracts: 'Contracts',
+    idvs: "Indefinite Delivery Vehicle",
+    grants: 'Grants',
+    direct_payments: 'Direct Payments',
+    loans: 'Loans',
+    other: 'Other'
+};
 
 export const awardTypeGroupLabels = {
     contracts: 'Contracts',
-    idvs: "Indefinite Delivery Vehicle",
```

Comment:
This is gonna change our google event logs? Just wondering why we're doing this.

Reply:
I also created an `analyticsAwardTypeGroupLabels` object and use that for logging to ensure our labels stay the same. See changes in [this file](https://github.com/fedspendingtransparency/usaspending-website/pull/1733/files/30bb529258dc6177700da69298fcc5355f3eb20e#diff-212b6f29b67fc6a1b9d64d785785eb9aR11). This is because it has been logging "Indefinite Delivery Vehicle" but the COVID page mocks use "Contract IDVs"